llama_stack/providers/inline/tool_runtime/rag/memory.py

```diff
@@ -65,11 +64,13 @@ async def insert(
         vector_db_id: str,
         chunk_size_in_tokens: int = 512,
     ) -> None:
+        converter: Converter = Converter.from_config(self.config.docling)
```

Comment:
My overall view about this is probably we should make `insert()` to be flexible enough that it supports broader input forms and Document conversion is a subset. This will make it easier when applications want to leverage typically structured data in memory as well.

Reply:
Since you're familiar with @feast-dev, you'll understand the analogue of the other sets of features that may be used for contextual awareness outside of just raw documents (e.g., a user's country of residence or shopping preferences).